test/ovsdb/ovn.dl.expected

```diff
@@ -0,0 +1,611 @@
+
+typedef integer      = bit<64>
+typedef uuid         = bit<128>
+typedef option_t<'A> = Some{x: 'A}
+                     | None
+
+input relation OVN_Northbound_NB_Global(
```

Comment:
The next step is to validate this automatically somehow.
For example by loading some data into this database.

Reply:
Yep, it's on the TODO list (#85) :)